extensions/git/src/api/git.d.ts

```diff
@@ -360,8 +360,8 @@ export const enum GitErrorCodes {
 	StashConflict = 'StashConflict',
 	UnmergedChanges = 'UnmergedChanges',
 	PushRejected = 'PushRejected',
-	ForcePushRejected = 'ForcePushRejected',
-	SaferForcePushRejected = 'SaferForcePushRejected',
+	ForcePushWithLeaseRejected = 'ForcePushWithLeaseRejected',
+	ForcePushWithLeaseIfIncludesRejected = 'ForcePushWithLeaseIfIncludesRejected',
```

Comment:
@lszomoru Don't we have to prepare an error message for the rejection of the `--force` push as before?
Also I wonder why you split `SaferForcePushRejected` into `ForcePushLease(IfIncludes)Rejected`.
The only difference between them is how safe they are. What you have to do when they are rejected are same. Do you have a future plan to split error messages for them?

Reply:
There is a concept of a push error handler where an extension can leverage the git extension API to register a push error handler. At the moment there are no push error handlers related to errors during force push, but in the future there might be so we want to make sure that the push error handler can distinguish between the two cases. Hope that makes sense.